cmd/dlv/cmds/commands.go

```diff
@@ -387,11 +387,11 @@ func dapCmd(cmd *cobra.Command, args []string) {
 		if workingDir != "" {
 			fmt.Fprintf(os.Stderr, "Warning: working directory ignored with dap; launch requests must specify full program path\n")
 		}
-		dlvArgs, targetArgs := splitArgs(cmd, args)
-		if len(dlvArgs) > 0 {
+		DlvArgs, TargetArgs := splitArgs(cmd, args)
```

Comment:
why the capitalization?

Reply:
Because I totally mixed that up :disappointed: . Changed